core/shared/src/main/scala/scalaz/zio/ZIO.scala

```diff
@@ -1102,6 +1102,20 @@ sealed trait ZIO[-R, +E, +A] extends Serializable { self =>
    * instance belongs (e.g. `IO.Tags.Succeed`).
    */
   def tag: Int
+
+  /**
+   * Exposes all parallel errors in a single call
+   *
+   */
+  final def parallelErrors: ZIO[R, List[E], A] =
```

Comment:
I'd change the `List[E]` here to `::[E]`.

Reply:
The problem is that E is invariant in `::` so you get this error: `Error:(398, 13) covariant type E occurs in invariant position in type => scalaz.zio.ZIO[R,::[E],A] of method parallelErrors final def parallelErrors: ZIO[R, ::[E], A]`

Reply:
```scala
final def parallelErrors[E1 >: E]: ZIO[R, ::E1, A] = ...
```

Reply:
👍

Reply:
@jdegoes @ghostdogpr using

 
```
 final def parallelErrors[E1 >: E]: ZIO[R, ::E1, A] = ...
```

I tried this with no success. Same error as

```
final def parallelErrors: ZIO[R, ::E1, A] = ...
```
Using list of course works. 

Any suggestions gentlemen? 

Also whats the benefit of using a cons `::E` over a `List`?

Thanks

Reply:
It is `::[E1]` instead of `::E1` ;)

Reply:
you are right I was blind to my mistake. Thanks for that @ghostdogpr